tests/math_translator_spec.js

```diff
@@ -0,0 +1,235 @@
+/*global describe, it*/
+
+const assert = require('assert');
+const MathTranslator = require('../lib/math-translator');
+
+const {
+    translateMath,
+    normalizeTranslatedMath,
+    MATH_RULES_LOCALES,
+} = MathTranslator;
```

Comment:
This can also be reduced to:
```
const {
    translateMath,
    normalizeTranslatedMath,
    MATH_RULES_LOCALES,
} = require('../lib/math-translator');
```